docs/en/docs/tutorial/sql-databases.md

```diff
@@ -97,18 +97,28 @@ We will create a FastAPI **dependency** with `yield` that will provide a new `Se
 
 Then we create an `Annotated` dependency `SessionDep` to simplify the rest of the code that will use this dependency.
 
-{* ../../docs_src/sql_databases/tutorial001_an_py310.py ln[25:30]  hl[25:27,30] *}
+{* ../../docs_src/sql_databases/tutorial001_an_py310.py ln[26:31]  hl[26:28,31] *}
 
 ### Create Database Tables on Startup
 
 We will create the database tables when the application starts.
 
-{* ../../docs_src/sql_databases/tutorial001_an_py310.py ln[32:37] hl[35:37] *}
+{* ../../docs_src/sql_databases/tutorial001_an_py310.py ln[34:42] hl[34:37, 42] *}
 
 Here we create the tables on an application startup event.
 
 For production you would probably use a migration script that runs before you start your app. 🤓
 
+
+/// warning
+
+The `@app.on_event("startup")` and `@app.on_event("shutdown")` decorators are **deprecated** as of FastAPI v0.103.0.
+
+Use the <a href="https://fastapi.tiangolo.com/advanced/events/#lifespan" class="external-link" target="_blank">`lifespan`</a> parameter in the `FastAPI` class instead for lifecycle management.
+
+///
+
+
```

Comment:
Update 2025-05-04:
I suggest to get rid of the warning above. It adds unnecessary overload to the `SQL Model` tutorial. The information about the deprecation of `@app.on_event` functionality is provided in the [Lifespan Event tutorial](https://fastapi.tiangolo.com/advanced/events/)